code/src/App.js

```diff
@@ -1,12 +1,28 @@
-import React from 'react'
-import data from './data.json'
+import React from "react";
+import data from "./data.json";
+import Album from "./Album/Album";
+import "./app.css";
+import Header from "./Header.js";
 
-console.log(data)
+console.log(data);
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
+    <div className="content">
+      <div className="header">
+        <Header />
+      </div>
+      <div className="albumList">
+        {data.albums.items.map(item => (
+          <Album
+            key={item.id}
+            item={item}
+            albumURL={item.external_urls.spotify}
+            artists={item.artists[0].name}
+            artistURL={item.artists[0].external_urls.spotify}
```

Comment:
Regarding the artists.
Since some of the albums have multiple artists you should make sure to show all of them.
Pass along the whole artists-array as a prop, so instead of artists and artistURL go for artists={album.artists}